neon_stt_plugin_nemo/__init__.py

```diff
@@ -26,60 +26,53 @@
 # NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
 # SOFTWARE,  EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-from inspect import signature
-from queue import Queue
+
+import speech_recognition as sr
+import numpy as np
+
+from streaming_stt_nemo import Model
 
 try:
-    from neon_speech.stt import StreamingSTT, StreamThread
```

Comment:
This try/except isn't needed anymore; STT can always be imported from `ovos_plugin_manager.templates.stt`